references/classification/utils.py

```diff
@@ -409,3 +409,22 @@ def reduce_across_processes(val):
     dist.barrier()
     dist.all_reduce(t)
     return t
+
+
+try:
+    import apex
+
+    apex_available = True
+except ImportError:
+    apex_available = False
+
+
+def master_params(optimizer):
+    """Generator to iterate over all parameters in the optimizer param_groups."""
+
+    if apex_available:
+        yield from apex.amp.master_params(optimizer)
```

Comment:
The implementation is exactly the same as below
https://github.com/NVIDIA/apex/blob/master/apex/amp/_amp_state.py#L67-L69.
In PyTorch domain lib, let's try to use PyTorch native as much as possible.

Reply:
Good catch! will remove this part